cernopendata/templates/cernopendata_records_ui/records/detail.html

```diff
@@ -112,182 +112,19 @@ <h2>Disclaimer</h2>
-      $ctrl.maxSize = 5;
-      $ctrl.files = [];
-      $ctrl.s_files = [];
-
-      var cleanStatus = Array.apply(null, Array($ctrl.totalItems)).map(function (_, i) {return true;});
-      $ctrl.itemsStatus = angular.copy(cleanStatus)
-
-      var selected_files = [];
-
-      $ctrl.start = ($ctrl.currentPage-1)*$ctrl.itemsPerPage;
-      $ctrl.end = ($ctrl.currentPage)*$ctrl.itemsPerPage;
-      for (var i=$ctrl.start; i < $ctrl.end; i++) {
-        $ctrl.itemsStatus[i] = false;
-      }
-
-      $ctrl.setPage = function (pageNo) {
-        $ctrl.currentPage = pageNo;
-      };
-
-      $ctrl.pageChanged = function() {
-        $ctrl.start = ($ctrl.currentPage-1)*$ctrl.itemsPerPage;
-        $ctrl.end = ($ctrl.currentPage)*$ctrl.itemsPerPage;
-        $ctrl.s_files = $ctrl.files.slice($ctrl.start, $ctrl.end);
-      };
-
-     $ctrl.getFiles = function(){
-        $http.get('/record/{{pid.pid_value}}/filepage/'+$ctrl.currentPage+'?group=1')
-          .then( function(files) {
-            $ctrl.index_files = files.data.index_files;
-            $ctrl.files = files.data.files;
-          })
-      }
-
-
-      $ctrl.pageChangedFetch = function(type = '') {
-        if (type == 'None') type = '';
-        $http.get('/record/{{pid.pid_value}}/filepage/'+$ctrl.currentPage+'?type='+type)
-          .then( function(files) {
-            $ctrl.files = files.data.files;
-            $ctrl.s_files = $ctrl.files;
-          })
-      };
-
-      $ctrl.setItemsPerPage = function(num) {
-        $ctrl.itemsPerPage = num;
-        $ctrl.currentPage = 1; //reset to first page
-      }
-}])
-
-
-recordApp.controller(
-  'downloadModalCtrl',
-  [
-    '$scope',
-    '$interpolate',
-    '$http',
-    '$uibModal',
-    function($scope,  $interpolate , $http, $uibModal) {
-      var $ctrl = this;
-      $ctrl.openModal = function (file, download_uri, filename) {
-        var modalInstance = $uibModal.open({
-          animation: $ctrl.animationsEnabled,
-          ariaLabelledBy: 'modal-title',
-          ariaDescribedBy: 'modal-body',
-          templateUrl: "/static/templates/downloadModal.html",
-          controller: 'downloadModalInstanceCtrl',
-          controllerAs: '$ctrl',
-          windowClass: 'download-modal',
-          resolve: {
-            file: function() {
-              return file;
-            },
-            filename: function() {
-              return filename;
-            },
-            download_uri: function() {
-              return download_uri
-            }
-          }
-        });
-      };
-    }
-  ]);
-
-recordApp.controller(
-  'downloadModalInstanceCtrl',
-  function ($scope, $uibModalInstance, file, filename, download_uri){
-    var $ctrl = this;
-
-    $ctrl.file = file;
-    $ctrl.filename = filename;
-    $ctrl.download_uri = download_uri;
-
-    $ctrl.closeModal = function () {
-        $uibModalInstance.close();
-    };
-  });
-
-
 recordApp.controller(
   'filePreviewModalCtrl',
```

Comment:
@diegodelemos I left this controller as you're working on it. Let me know if I should get rid of it or you do.

Reply:
You can get rid of it :)

Reply:
Done. After this PR all angular code is gone.